src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py

```diff
@@ -26,11 +25,17 @@ def __init__(self, config):
         BaseWorkerThread.__init__(self)
         self.config = config
 
+        self.tasksCPU = ['Processing', 'Production']
+        self.tasksIO = ['Merge', 'Cleanup', 'Harvesting', 'LogCollect', 'Skim']
+        self.minCPUSlots = 50
+        self.minIOSlots = 25
+
         # get dashboard url, set metric columns from config
         self.dashboard = config.AgentStatusWatcher.dashboard
         self.siteStatusMetric = config.AgentStatusWatcher.siteStatusMetric
         self.cpuBoundMetric = config.AgentStatusWatcher.cpuBoundMetric
         self.ioBoundMetric = config.AgentStatusWatcher.ioBoundMetric
```

Comment:
how often this metric changes, maybe we can set the default value, just for the information purpose.

Reply:
I think it hasn't changed for a year or two. I'm not sure I got your comment, set the default value where? ReqMgrAux? IMO it's not really needed.